data/successors/successor weapons.txt

```diff
@@ -0,0 +1,343 @@
+		"piercing" 0.05
+		"cluster"
+	description `Unable to access large quantities of the high-quality weaponry produced by the High Houses, the People's Houses have had to work with what's available to them: mostly the relatively low-powered pulse lasers produced independently in Successor space. This pulse laser has had its safeties bypassed and its optics upgraded, allowing it to do more damage to shields at a lower power cost in exchange for producing substantially more heat.`
+
+outfit "Overcharged Laser Turret"
+	category "Turrets"
+	cost 136000
+	thumbnail "outfit/overcharged pulse laser turret"
+	licenses
+		"Successor"
+	"mass" 24
+	"outfit space" -24
+	"weapon capacity" -24
+	"turret mounts" -1
+	weapon
+		sprite "projectile/successor pulse laser"
+			"start frame" 2
+			"frame rate" 8
+			"rewind"
+		"hardpoint sprite" "hardpoint/laser turret"
+		sound "laser"
+		"fire effect" "moonbeam fleck" 3
+		"hit effect" "electron impact"
+		"inaccuracy" 5.5
+		"velocity" 490
+		"lifetime" 1
+		"turret turn" 3.5
+		"burst count" 4
+		"burst reload" 1
+		"reload" 7
+		"firing energy" 12.6
+		"firing heat" 43.05
+		"shield damage" 19.6
+		"hull damage" 18.667
+		"piercing" 0.05
+		"cluster"
+	description `Hand-crafted by the more colorful members of the People's Houses, this bespoke overcharged Pulse Laser turret has had even more power pumped into its laser generator compared to the fixed version, producing a truly prodigious amount of heat for a weapon of its size. Despite its inelegant design, it is nonetheless a fearsome weapon, especially considering that it retains the ability to partially punch through shielding.`
+
+outfit `"Sjeja" Kinetic Lance`
+	category "Secondary Weapons"
+	cost 3500000
+	licenses
+		"High Houses"
+	thumbnail "outfit/sjeja kinetic lance"
+	"mass" 60
+	"outfit space" -60
+	"weapon capacity" -60
+	"energy capacity" 1900
+	"gun ports" -1
+	"spike capacity" 20
+	weapon
+		sprite "projectile/sjeja kinetic lance"
+			"frame rate" 4
+			"random start frame"
+		sound "typhoon"
+		ammo "High-Velocity Spike"
+		icon "icon/hv spike"
+		"fire effect" "moonbeam fleck" 20
+		"fire effect" "electron impact"
+		"hit effect" "torpedo hit"
+		"die effect" "tiny explosion"
+		"cluster"
+		"velocity" 320
+		"lifetime" 8
+		"reload" 160
+		"firing force" 1200
+		"firing energy" 1900
+		"firing ion" 3.0
+		"firing heat" 4000
+		"shield damage" 1336
+		"hull damage" 1920
+		"piercing" .25
+		"hit force" 3750
+		"penetration count" 5
+	description "The largest and most powerful of the Successors' modern weapons, this massive coilgun is elegant in its simplicity, cleanly perforating targets at extreme range. Because the bank of exotic electromagnets it employs requires an unusually high burst of power when firing, the weapon comes with integrated battery banks in order to supply it with a large enough reserve of electricity to function."
+
+outfit "High-Velocity Spike"
+	category "Ammunition"
+	licenses
+		"High Houses"
+	cost 4000
```

Comment:
That is _expensive_.
Should the cost be lowered, even slightly?

Reply:
I'm not particularly attached to this price, but right now the cost-per-second to fire the "Sjeja" is more or less in line with other secondary weapons at the same tier: 1500 credits/sec is the same as the Bunrodea Swarm Pod, 1440 for the Wanderer Thunderhead, etc.
I suppose it might make sense for this to be a little cheaper given it's not an explosive weapon.

Reply:
Maybe down to 3500? Though I suppose this does do more damage per second then the other two, so it should be expensive.